src/main/java/com/github/stefanbirkner/systemlambda/SystemLambda.java

```diff
@@ -1159,7 +1179,7 @@ private String format(
 		 * environment variables map. It fails if your {@code SecurityManager} forbids
 		 * such modifications.
 		 * @param statement an arbitrary piece of code.
-		 * @throws Exception any exception thrown by the statement.
```

Comment:
not related to my change but I noticed a couple of warnings during the build related to the exception documentation being missing.